features/rhelah/atomic/rpm-ostree_bad_ca_cert.feature

```diff
@@ -0,0 +1,15 @@
+Feature: tests whether RH CA pinning works for rpm-ostree
+    It is critical for security that we're "pinned" to the RHN CA cert. If another server certificate is presented we should error out with something like Unacceptable TLS certificate.
+
+Scenario: 0. Host provisioned and subscribed
+    When "all" host is auto-subscribed to "prod"
+    Then subscription status is ok on "all"
+    and "1" entitlement is consumed on "all"
```

Comment:
However usage of `@host_subscribed` subscribes host to the stage environment where http connection is used by default. So this must be fixed in `atomic-ci-jobs` repo as well.